functests/3_performance_status/status.go

```diff
@@ -34,6 +37,29 @@ var _ = Describe("Status testing of performance profile", func() {
 		nodeAnnotationDesiredConfig := "machineconfiguration.openshift.io/desiredConfig"
 		nodeLabel := map[string]string{fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerCNF): ""}
 
+		It("Tuned status field tied to Performance Profile", func() {
```

Comment:
Can we please add test_id here as OCP-30894

Reply:
@gsr-shanks  done.